smoke/IotEdgeQuickstart/Program.cs

```diff
@@ -114,6 +114,9 @@ class Program
         [Option("-l|--deployment <filename>", Description = "Deployment json file")]
         public string DeploymentFileName { get; } = Environment.GetEnvironmentVariable("deployment");
 
+        [Option("-tw|--twin_test <filename>", Description = "A file with Json content to set desired property and check reported property in a module.")]
+        public string TwinTestFileName { get; } = null;
```

Comment:
Should it be consistent to use string.Empty instead of null?

Reply:
would you help me to understand when to use "-" and "_" in the parameter option?

Reply:
Regarding 1st somment, null is consistent with the expected result of Deployment file name. The GetEnvironmentVariable returns null. Since I won't put an environment variable for that, I'm using directly null. And we are not passing the string to quickStart. We are passing Option, so I think null should be good for this case. 

Question 2 we don't have a standard for that today. If you take a look into bellow parameters (For CA Cert, etc, it uses _). We can change everything to follow 1 pattern in a different CR.